### PR TITLE
Make EthLog initiator respect toBlock, fromBlock, and topic params

### PR DESCRIPTION
**Pull request by spooktheducks**
https://www.pivotaltracker.com/n/projects/2129823/stories/163464413

### CODE REVIEW

File: core/store/models/job_spec.go [hunk 1]
```diff
@@ -263,6 +263,33 @@ type InitiatorParams struct {
...
+
+func (t Topics) Scan(value interface{}) error {
+	jsonStr, ok := value.(string)
+	if !ok {
+		return fmt.Errorf("Unable to convert %v of %T to Topics", value, value)
+	}
+
+	err := json.Unmarshal([]byte(jsonStr), &t)
+	if err != nil {
+		return fmt.Errorf("Unable to convert %v of %T to Topics: %v", value, value, err)
```
Comment (felder-cl) [inline core/store/models/job_spec.go]: This is good, but maybe errors.Wrap(..) would be better?
https://godoc.org/github.com/pkg/errors#Wrapf

File: core/store/models/log_events.go [hunk 2]
```diff
@@ -88,30 +88,43 @@ func TopicFiltersForRunLog(logTopics []common.Hash, jobID *ID) [][]common.Hash {
...
+		} else if from != nil && i.InitiatorParams.FromBlock != nil {
+			from = utils.MaxBigs(from, i.InitiatorParams.FromBlock.ToInt())
+		}
+		q := ethereum.FilterQuery{
+			FromBlock: from,
+			ToBlock:   i.InitiatorParams.ToBlock.ToInt(),
+			Addresses: utils.WithoutZeroAddresses([]common.Address{i.Address}),
+			Topics:    i.Topics,
+		}
+		logger.Warnf("FilterQueryFactory %+v", q)
```
Comment (felder-cl) [inline core/store/models/log_events.go]: Curious, why is this considered a Warning(..) as opposed to Info(..)?

Comment (felder-cl) [reply core/store/models/log_events.go]: As far as I can see, this case isn't coverd by tests.  Anyway this could be improved?

Comment (spooktheducks) [reply core/store/models/log_events.go]: Agreed, ought to be, but wanted some input about the team's testing strategy before arriving at my own opinion.

File: core/store/models/log_events.go [hunk 3]
```diff
@@ -88,30 +88,42 @@ func TopicFiltersForRunLog(logTopics []common.Hash, jobID *ID) [][]common.Hash {
...
+			Addresses: utils.WithoutZeroAddresses([]common.Address{i.Address}),
+			Topics:    i.Topics,
+		}
+		return q, nil
+
 	case InitiatorRunLog:
 		topics := []common.Hash{RunLogTopic20190207withoutIndexes, RunLogTopic20190123withFullfillmentParams, RunLogTopic0original}
 		filters := TopicFiltersForRunLog(topics, i.JobSpecID)
-		return newInitiatorFilterQuery(i, from, filters), nil
+		return ethereum.FilterQuery{
```
Comment (j16r) [inline core/store/models/log_events.go]: Looks like you could assemble this filter query at the top of the function and just fill in `ToBlock` under `InitiatorEthLog`.

**Pull request merged**